api/src/main/java/org/openmrs/scheduler/Schedule.java

```diff
@@ -231,21 +226,6 @@ public void setRepeatInterval(long repeatInterval) {
 	}
 	
 	/**
-	 * Get the date format used to set the start time.
-	 */
-	public String getDateFormat() {
```

Comment:
It could potentially cause module compile errors, though, no?

Or, if we aren't careful and run a module against the latest version of core without testing that it compiles, could result in runtime errors?

That being said, these do look like little-used methods.